uwblueprint/marillac-place
# Frontend setup - hitting apis and add chakra ui

## Notion ticket link

[Frontend setup - hitting apis and add chakra ui](https://www.notion.so/uwblueprintexecs/Frontend-setup-hitting-apis-and-add-chakra-ui-1292340de3e24cebbc7217287d21a0da)



Upgrade React Scripts to v5 and adding Chakra UI v 2.8.2 to front end 
## Implementation description
* Upgraded React Scripts 
* Added Chakra UI package to package.json
* Fixed React Router Bugs, fixed naming bugs 
* Broke Authentication (will fix in future ticket)


## Steps to test
1. Pull branch changes
2. Run Docker compose 
3. See if front end works



## What should reviewers focus on?
* AUTH IS DISABLED because we removed jsonwebtoken for react scripts 5


## Checklist
- [x ] My PR name is descriptive and in imperative tense
- [ x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
